docs/src/components/MDXHeading.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import {
+  Heading,
+  Icon,
+  Tooltip
+} from '@innovaccer/design-system';
+
+const MDXHeading=({size,headingInfo })=>{
+  const refHeading= React.createRef();
+  const [isHovered, setIsHovered] = useState(false);
+  const [tooltipName, setTooltipName] = useState("Copy");
+  const [isTooltipActiveHeading , setTooltipActiveHeading] = useState(false);
+  const HoverHandler=(type , )=>{
+    if(type ==="Enter"){
```

Comment:
function name "HoverHandler" should follow this regular expression ^[a-z][a-zA-Z0-9]*$